x-pack/elastic-agent/docs/elastic-agent-command-line.asciidoc

```diff
@@ -235,9 +353,92 @@ Set log level to INFO.
 
 [source,shell]
 ----
-elastic-agent run -v --path.logs path/to/my/logs
+elastic-agent run -c myagentconfig.yml
+----
+
+++++
+<hr>
+++++
+
+[discrete]
+[[elastic-agent-uninstall-command]]
+== elastic-agent uninstall
+
+Permanently uninstall {agent} from the system.
+
+[discrete]
+=== Synopsis
+
+[source,shell]
+----
+elastic-agent uninstall [--force] [--help] [global-flags]
+----
+
+[discrete]
+=== Options
+
+`--force`::
+Uninstall {agent} and do not prompt for confirmation. This flag is helpful
+when using automation software or scripted deployments.
+
+`--help`::
+Show help for the `uninstall` command.
+
+{global-flags-link}
+
+[discrete]
+=== Examples
+
+[source,shell]
+----
+elastic-agent uninstall
+----
+
+++++
+<hr>
+++++
+
+[discrete]
+[[elastic-agent-upgrade-command]]
+== elastic-agent upgrade
+
+Upgrade the currently running {agent} to the specified version.
+
+[discrete]
+=== Synopsis
+
+[source,shell]
+----
+elastic-agent upgrade <version> [--source-uri <string>] [--help] [flags]
 ----
 
+[discrete]
+=== Options
+
+`version`::
+The version of {agent} to upgrade to.
+
+`--source-uri <string>`::
+The source URI to download the new version from.
+
+//REVIEWERS: Is the source URI required or does it default to a specific URI?
+//Can't really test this...yet. Will the URI just be the download URL?
```

Comment:
It is not required and in normal usage, it is not needed. By default Agent will use the Elastic Artifacts URL.